src/librustc_mir/shim.rs

```diff
@@ -250,6 +243,27 @@ fn build_drop_shim<'tcx>(tcx: TyCtxt<'tcx>, def_id: DefId, ty: Option<Ty<'tcx>>)
     body
 }
 
+fn new_body<'tcx>(basic_blocks: IndexVec<BasicBlock, BasicBlockData<'tcx>>,
```

Comment:
please use block indentation for the arguments in this file for consistency (the other function in this file use it, too)